src/js/components/TableCell/index.d.ts

```diff
@@ -2,7 +2,7 @@ import * as React from 'react';
 import { BoxTypes } from '../Box';
 
 export interface TableCellProps {
-  plain?: boolean;
+  plain?: boolean | 'noPad';
```

Comment:
```suggestion
  plain?: boolean;
```
We should not expose `noPad`